pkg/virt-handler/cmd-client/client.go

```diff
@@ -658,3 +659,26 @@ func (c *VirtLauncherClient) Exec(domainName, command string, args []string, tim
 
 	return exitCode, stdOut, err
 }
+
+func (c *VirtLauncherClient) GuestPing(domainName string, timeoutSeconds int32) (int, error) {
+	request := &cmdv1.GuestPingRequest{
+		DomainName:     domainName,
+		TimeoutSeconds: timeoutSeconds,
+	}
+	exitCode := -1
+
+	ctx, cancel := context.WithTimeout(
+		context.Background(),
+		// we give the context a bit more time as the timeout should kick
+		// on the actual execution
+		time.Duration(timeoutSeconds)*time.Second+shortTimeout,
+	)
+	defer cancel()
+
+	_, err := c.v1client.GuestPing(ctx, request)
```

Comment:
Shouldn't we still have a response here with an potential errorCode/message?
If we don't use them we can remove them from the proto or we use them to pass the exitCode along.

Reply:
The limited info about `guest-ping` suggests that only the `err` is important to us
https://wiki.qemu.org/Features/GuestAgent